dac/backend/src/main/java/com/dremio/dac/support/SupportService.java

```diff
@@ -57,13 +57,19 @@
   String CLUSTER_IDENTITY = "clusterIdentity";
   String DREMIO_EDITION = "dremioEdition";
 
+  /**
+   * Acquires the current cluster identity.  The cluster identity is used to identify
```

Comment:
Use the `<p>` tag to separate paragraphs.

Reply:
done